packages/samples/test/output/versioning/openapi.v1.json

```diff
@@ -84,7 +84,7 @@
             "items": {
               "$ref": "#/components/schemas/Library.PetToy"
             },
-            "x-cadl-name": "Library.PetToy[]"
+            "x-cadl-name": "Array<Library.PetToy>"
```

Comment:
Is this expected?

Reply:
It is because Array are just model now.  We could have the getTypeName have a special case for array to show it back as `[]`

Reply:
I think we should special case it since you can't refer to it as `Array<Library.PetToy>` in Cadl stil, can you?

Reply:
you can 

https://cadlplayground.z22.web.core.windows.net/prs/571/?c=bW9kZWwgRm9vIHsNCg0KICBhOiBBcnJheTxzdHJpbmc%2BOw0KfQ%3D%3D

Reply:
so actually I already made that special case... but seems like it isn;t working here, will investigate

Reply:
hhm seems to be related to the versioning, might be a bug with projection and the type system